docs/usage/elasticsearch_container.md

```diff
@@ -16,13 +16,21 @@ ElasticsearchContainer container = new ElasticsearchContainer("docker.elastic.co
 // Start the container. This step might take some time...
 container.start();
 
-// Do whatever you want here.
+// Do whatever you want with the rest client ...
 final CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
 credentialsProvider.setCredentials(AuthScope.ANY, new UsernamePasswordCredentials("elastic", "changeme"));
-RestClient client = RestClient.builder(HttpHost.create(container.getHttpHostAddress()))
+RestClient restClient = RestClient.builder(HttpHost.create(container.getHttpHostAddress()))
         .setHttpClientConfigCallback(httpClientBuilder -> httpClientBuilder.setDefaultCredentialsProvider(credentialsProvider))
         .build();
-Response response = client.performRequest("GET", "/");
+Response response = restClient.performRequest("GET", "/");
+
+// ... or the transport client
+TcpHost tcpHost = container.getTcpHost();
+TransportAddress transportAddress = new TransportAddress(new InetSocketAddress(tcpHost.getHostname(), tcpHost.getPort()));
```

Comment:
Thanks for your flexibility on making this change @reda-alaoui.

We just need these docs to be updated, then I think we're good.

Reply:
Donc updated :+1: